osu/map.py

```diff
@@ -52,9 +53,26 @@ def files(self):
 
     def zip(self):
         map_pack_file = MAP_PACKS_FOLDER.joinpath(f"{uuid.uuid4()}.zip")
-
+        print(map_pack_file)
         files = self.files()
         FileTools.zip_files(files, to_file=map_pack_file)
-        FileTools.delete_files(files)
 
         return map_pack_file
+
+    def upload(self, dbx_path: str) -> str | None:
+        zf = self.zip()
+        path = zf.name
+        dbx = dropbox.Dropbox(DROPBOX_ACCESS_TOKEN)
+
+        try:
+            with open(zf, "rb") as f:
+                dbx.files_upload(f.read(), dbx_path + path)
```

Comment:
if you have to combine the same thing multiple times it should probs be a variable

Reply:
ok dad